packages/terra-layout/src/utilities.js

```diff
@@ -0,0 +1,31 @@
+import breakpoints from 'terra-responsive-element/lib/breakpoints.scss';
```

Comment:
Nit: Update name to LayoutUtils to follow naming pattern of other packages.

Reply:
+1